torch/distributions/distribution.py

```diff
@@ -35,6 +35,27 @@ def __init__(self, batch_shape=torch.Size(), event_shape=torch.Size(), validate_
                 if not constraint.check(getattr(self, param)).all():
                     raise ValueError("The parameter {} has invalid values".format(param))
 
+    def expand(self, batch_shape, instance=None):
```

Comment:
Consider renaming `instance` to `_instance` and document generally only used by derived classes that need to override the `.expand()` method. It's kind of gross, but I don't see a simpler solution.